src/app/qgsmeasuredialog.h

```diff
@@ -98,6 +98,15 @@ class APP_EXPORT QgsMeasureDialog : public QDialog, private Ui::QgsMeasureBase
     //! indicates whether we're measuring distances or areas
     bool mMeasureArea = false;
 
+    //! Indicates whether the user chose "Map units" instead of directly selecting a unit
+    bool useMapUnits = false;
```

Comment:
QGIS code convention is that member variables all have a "m" prefix, so this should be `mUseMapUnits` (and `mMapDistanceUnits`, etc)

Reply:
Done